tests/instrumentation-framework/src/test/resources/testOpenTracingPolyglot/test.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "OpenTracing polyglot test",
+  "environment": {
+    "dockerCompose": "docker-compose.yml",
+    "initWaitSeconds": "20",
+    "type": "APM",
+    "apmAddress": "172.16.158.1"
+  },
+  "tests": [
+    {
+      "description" : "Docker compose JBoss Ticker Monster",
+      "skip": false,
+      "action": "action.sh",
+      "scriptServiceName": "nodejs",
+      "afterActionWaitSeconds": "20",
+      "verify": {
+        "jsonPath": [
+          {
+            "path": "$.length()",
+            "result": 1
```

Comment:
Once both fragments recorded, it would be good if we could compare the traceId field in both to make sure they are the same, which should indicate the state has been propagated.